--- anytopnm.org	2005-12-31 07:57:24.000000000 +0100
+++ anytopnm	2006-01-01 12:09:15.000000000 +0100
@@ -99,9 +99,9 @@
     giftopnm "$file"
     ;;
 
-    *TIFF* )
-    tifftopnm "$file"
-    ;;
+#    *TIFF* )
+#    tifftopnm "$file"
+#    ;;
 
     *IFF*ILBM* )
     ilbmtoppm "$file"
@@ -241,6 +241,9 @@
         *.mda | *.mdp )
         mdatopbm -d -- "$file"
         ;;
+        *.nef | *.crw | *.cr2 | *.NEF | *.CRW | *.CR2 )
+        dcraw -2 -c -w "$file"
+        ;;
         * )
         echo "$0: unknown file type: $filetype" 1>&2
         exit 1
